data/scripting/editor/editor_controls.lua

```diff
@@ -14,9 +14,7 @@ local r = {
       p(
          -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
```

Comment:
May I remove "Localize, but do not change the key." from all of these? The key hasn't been part of the translatable text for a long time now…